app/views/procedures/new.html.erb

```diff
@@ -0,0 +1,7 @@
+<div class="mt-24 md:w-2/3 w-full">
+  <h1 class="font-bold text-4xl">New procedure</h1>
+
+  <%= render "form", procedure: @procedure %>
+
+  <%= link_to 'Back', categories_path, class: "ml-2 rounded-lg py-3 px-5 bg-gray-100 inline-block font-medium" %>
+</div>
```

Comment:
![image](https://user-images.githubusercontent.com/38635158/205610246-3ef295ce-f4d6-4ff9-8ee0-e1a4fe31f1e2.png)

- [ ] I am getting an alert error when ı creating a new transaction for a better user experience kindly fix it. It also appear in your presentation video.
- [ ] It happens when clicking on the back button too.
- [ ] It happens when ı logout and sign in again too.

Reply:
Done @ahmetbozaci